### ISSUE TITLE
Incorrect formatting for properties with by keyword

### ISSUE THREAD

**Issue by scottdweber** [pinterest/ktlint#1162]
Given the following code:
```kotlin
private val aSampleValue: SampleValue
    by lazy {
        SampleValue()
    }
```

## Expected Behavior
Leave the code unchanged. This is how Android Studio formats this code when using the official Kotlin style guide.

## Observed Behavior
0.39.0 formats this way, which isn't _terrible_, but also doesn't match official Kotlin style guide:
```kotlin
private val aSampleValue: SampleValue
by lazy {
    SampleValue()
}
```

0.40.0 and 0.41.0 formats this way, which is quite unpleasant:
```kotlin
private val aSampleValue: SampleValue
    by lazy {
    SampleValue()
}
```

## Your Environment
* Version of ktlint used: 0.39.0, 0.40.0, 0.41.0
* org.jlleitschuh.gradle:ktlint-gradle:10.1.0
* Version of Gradle used (if applicable): 6.6.1
* Operating System and version: macOS 11.4
* Android Studio: 4.2.1
